frb_codegen/Cargo.toml

```diff
@@ -29,6 +29,8 @@ extend = "1.1.2"
 itertools = "0.10.3"
 lazy_static = {workspace = true}
 log = "0.4"
+fern = { version = "0.6.0", features = ["date-based"] }
```

Comment:
nit: We already have `env_logger` and has setup that one; now we also use `fern`. Will we have conflicts, or can we reduce such two duplicated crates into one?